src/sentry/conf/server.py

```diff
@@ -3442,3 +3442,14 @@ class TopicDefinition(TypedDict):
 # The Redis cluster to use for monitoring the health of
 # Celery queues.
 SENTRY_QUEUE_MONITORING_REDIS_CLUSTER = "default"
+
+# This is a mapping between the various processing stores,
+# and the redis `cluster` they are using.
+# This setting needs to be appropriately synched across the various deployments
+# for automatic backpressure management to properly work.
+SENTRY_BACKPRESSURE_STORES = {
+    "attachments": "rc-short",
+    # "processing": "processing",
+    "locks": "default",
+    "post_process_locks": "default",
```

Comment:
How about we make this a list so that we can just list higher-level scopes (neither clusters nor consumers), such as:

```suggestion
    "processing": ["default"]  # includes locks and post-process locks
```

You'd then set the key unhealthy if any of the clusters is unhealthy.

Reply:
I would rather not do that. While these things can always go out of sync, I would rather the association which consumer accesses which store to be defined in code, rather than having to sync a bunch of settings between all the different deployments.